File: VB/WebApplication1/Controllers/HomeController.vb

Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc
Imports DevExpress.Web.Mvc
Imports WebApplication1.WebApplication1.Models

Namespace WebApplication1.Controllers
    Public Class HomeController
        Inherits Controller

        Public Function Index() As ActionResult
            Return View()
        End Function
        Public Function GridViewPartial(ByVal andOrValue? As Boolean) As ActionResult
            ViewBag.Licenses = LicensesDataProvider.Licenses
            ViewBag.andOrValue = andOrValue
            Return PartialView("_GridViewPartial", CustomersDataProvider.Customers)
        End Function
    End Class
End Namespace

File: CS/WebApplication1/Controllers/HomeController.cs
using WebApplication1.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using DevExpress.Web.Mvc;

namespace WebApplication1.Controllers {
    public class HomeController : Controller {
        public ActionResult Index() {
            return View();
        }
        public ActionResult GridViewPartial(bool? andOrValue) {
            ViewBag.Licenses = LicensesDataProvider.Licenses;
            ViewBag.andOrValue = andOrValue;
            return PartialView("_GridViewPartial", CustomersDataProvider.Customers);
        }
    }
}